dephell/converters/setuppy.py

```diff
@@ -34,6 +35,16 @@
 logger = getLogger('dephell.converters.setuppy')
 
 
+class SortedDictDefaultList(SortedDict):
```

Comment:
where should this type go?  It is beneficial even if `OrderedDict` is used.

Reply:
It's the best place, I think :+1: